packages/backend/src/services/connection.ts

```diff
@@ -0,0 +1,89 @@
+import { IFlowCollabRole } from '@plumber/types'
+
+import { NotFoundError, Transaction } from 'objection'
+
+import Context from '@/types/express/context'
+
+type GetConnectionParams = {
+  context: Context
+  connectionId: string
+  flowId: string
+  requiredRole: IFlowCollabRole
+  role?: IFlowCollabRole
+  trx?: Transaction
+}
```

Comment:
- flowId should always be defined here
- requiredRole not needed since it's always viewer
- role change to includeOwnConnections: boolean which is when 'role' === 'owner'